index.html

```diff
@@ -263,19 +263,43 @@ <h3 class="box3">
             <li class="lang">
                 <p>
                     FrameWorks
-                   </p>    
-                     <img class="imglang-title" src="images/arrow-side.png" alt="">
+                </p>
+                <img class="imglang-title" src="images/arrow-side.png" alt="">
             </li>
             <li class="lang-line"></li>
             <li class="lang">
                 <p>
                     Skills
-                   </p>    
-                     <img class="imglang-title" src="images/arrow-side.png" alt="">
+                </p>
+                <img class="imglang-title" src="images/arrow-side.png" alt="">
             </li>
             <li class="lang-line"></li>
         </ul>
-            
+
+    </section>
+    <section class="contact-me">
+        <div class="bg-image">
+            <h1 class="contact-title">Contact me </h1>
+            <p class="contact-text">
+                If you have an application you are interested in developing, a feature that you need built or a project
+                that
+                needs coding. I’d love to help with it
+            </p>
+            <form action="https://formspree.io/f/xzbodpro" method="post" class="contact-form">
+                <input type="text" name="name" id="username" placeholder="Your Name" required>
```

Comment:
- [ ] Please, consider, that the name input should has `maxlength="30"` attribute.